CSharpSourceCode/CampaignSupport/WanderersController.cs

```diff
@@ -0,0 +1,85 @@
+﻿using HarmonyLib;
+using System.Collections.Generic;
+using System.Linq;
+using TaleWorlds.CampaignSystem;
+using TaleWorlds.CampaignSystem.Actions;
+using TaleWorlds.CampaignSystem.SandBox.CampaignBehaviors;
+using TaleWorlds.Core;
+using TOW_Core.Utilities.Extensions;
+
+namespace TOW_Core.CampaignSupport
+{
+    public class WanderersController : CampaignBehaviorBase
```

Comment:
please choose a fitting name for your class like the rest of the classes that inherit the CampaignBehaviorBase class